modules/aws/vpc.go

```diff
@@ -143,6 +153,60 @@ func GetSubnetsForVpcE(t testing.TestingT, vpcID string, region string) ([]Subne
 	return subnets, nil
 }
 
+// GetTagsForVpc gets the tags for the specified VPC.
+func GetTagsForVpc(t testing.TestingT, vpcID string, region string) map[string]string {
+	tags, err := GetTagsForVpcE(t, vpcID, region)
+	if err != nil {
+		t.Fatal(err)
+	}
+	return tags
+}
+
+// GetTagsForVpcE gets the tags for the specified VPC.
+func GetTagsForVpcE(t testing.TestingT, vpcID string, region string) (map[string]string, error) {
+	client, err := NewEc2ClientE(t, region)
+	if err != nil {
+		return nil, err
+	}
+
+	vpcResourceTypeFilter := ec2.Filter{Name: &vpcResourceTypeFilterName, Values: []*string{&vpcResourceTypeFilterValue}}
+	vpcResourceIdFilter := ec2.Filter{Name: &vpcResourceIdFilterName, Values: []*string{&vpcID}}
+	tagsOutput, err := client.DescribeTags(&ec2.DescribeTagsInput{Filters: []*ec2.Filter{&vpcResourceTypeFilter, &vpcResourceIdFilter}})
+	if err != nil {
+		return nil, err
+	}
+
+	tags := map[string]string{}
+	for _, tag := range tagsOutput.Tags {
+		tags[aws.StringValue(tag.Key)] = aws.StringValue(tag.Value)
+	}
+
+	return tags, nil
+}
+
+// GetTagValueForVpc gets the value of the tag for the specified VPC tag.
+func GetTagValueForVpc(t testing.TestingT, vpcID string, region string, tagKey string) string {
+	tagValue, err := GetTagValueForVpcE(t, vpcID, region, tagKey)
+	if err != nil {
+		t.Fatal(err)
+	}
+	return tagValue
+}
+
+// GetTagValueForVpcE gets the value of the tag for the specified VPC tag.
+func GetTagValueForVpcE(t testing.TestingT, vpcID string, region string, tagKey string) (string, error) {
+	tags, err := GetTagsForVpcE(t, vpcID, region)
+	if err != nil {
+		return "", err
+	}
+
+	if tagValue, tagExists := tags[tagKey]; tagExists {
+		return tagValue, nil
+	} else {
+		return "", fmt.Errorf("VPC does not have any Tag with a key of %s", tagKey)
+	}
+}
```

Comment:
Hmm I'm not sure this function passes the **complexity** litmus test, and I also am not sure this fits the bill for *completeness* either. I'd be inclined to remove this so as not to bloat our API.

Reply:
OK. No worries. Removed both.